### ISSUE TITLE
allowbackup

### ISSUE THREAD

**Issue by mesterj** [miteshpithadiya/SearchableSpinner#102]
Please turn off autobackup in manifest. It is really a problem.

**Comment by welshrat77** [comment]
Yes same here, I need to turn off autoBackup in my project which means I can't use this library

EDIT - fixed by overriding however would be nice to not have to given this is a library, thanks for creating it regardless, useful library
